extra-cpp-bindings/src/lib.rs

```diff
@@ -175,48 +184,87 @@ mod ffi {
             api_key: String,
         ) -> Result<Vec<RawTxDetail>>;
+        /// returns the ERC20 transfers of a given address of a given contract.
+        /// (address can be empty if option is ByContract)
+        /// default option is by address
+        /// The API key can be obtained from https://cronoscan.com
         pub fn get_erc20_transfer_history_blocking(
             address: String,
             contract_address: String,
             option: QueryOption,
             api_key: String,
         ) -> Result<Vec<RawTxDetail>>;
+        /// returns the ERC721 transfers of a given address of a given contract.
+        /// (address can be empty if option is ByContract)
+        /// default option is by address
+        /// The API key can be obtained from https://cronoscan.com
         pub fn get_erc721_transfer_history_blocking(
             address: String,
             contract_address: String,
             option: QueryOption,
             api_key: String,
         ) -> Result<Vec<RawTxDetail>>;
-        /// BlockScout API
+        /// given the BlockScout REST API base url and the account address (hexadecimal),
+        /// it will return the list of all owned tokens
+        /// (ref: https://cronos.org/explorer/testnet3/api-docs)
         pub fn get_tokens_blocking(
             blockscout_base_url: String,
             account_address: String,
         ) -> Result<Vec<RawTokenResult>>;
+        /// given the BlockScout REST API base url and the account address (hexadecimal; required)
+        /// and optional contract address (hexadecimal; optional -- it can be empty if the option is ByAddress),
+        /// it will return all the token transfers (ERC20, ERC721... in the newer BlockScout
+        /// releases, also ERC1155)
+        /// (ref: https://cronos.org/explorer/testnet3/api-docs)
+        /// NOTE: QueryOption::ByContract is not supported by BlockScout
         pub fn get_token_transfers_blocking(
             blockscout_base_url: String,
             address: String,
             contract_address: String,
             option: QueryOption,
         ) -> Result<Vec<RawTxDetail>>;
-        /// Crypto.com Pay API
+        /// given the BlockScout REST API base url and the contract address (hexadecimal),
+        ///
+        /// page: A nonnegative integer that represents the page number to be used for
+        /// pagination. 'offset' must be provided in conjunction.
+        ///
+        /// offset: A nonnegative integer that represents the maximum number of records to
+        /// return when paginating. 'page' must be provided in conjunction. token ids.
+        ///
+        /// It will return the list of owners and balances (sorting from largetst to smallest), but no
+        ///
+        /// (ref: https://cronos.org/explorer/api-docs#token)
+        pub fn get_token_holders(
+            blockscout_base_url: String,
+            contract_address: String,
+            page: String,
+            offset: String,
```

Comment:
could page and offset be unsigned integers instead of strings?

Reply:
done